src/main/scala/cromwell/logging/TerminalLayout.scala

```diff
@@ -26,7 +29,7 @@ class TerminalLayout extends LayoutBase[ILoggingEvent] {
      */
     val prefix = if (event.getMessage == "{}") s"[${TerminalUtil.highlight(129, "*")}] " else ""
```

Comment:
Not specific to this but as a general rule I think commenting things out so they don't get forgotten needs an extremely strong argument to justify.